res/css/views/rooms/_EditMessageComposer.scss

```diff
@@ -16,39 +16,41 @@ limitations under the License.
 */
 
 .mx_EditMessageComposer {
+    background-color: $header-panel-bg-color;
+    position: absolute;
```

Comment:
it's not immediately obvious what this will be relative to - a quick comment would be appreciated.